app/src/main/java/ch/epfl/culturequest/ArtDescriptionDisplayActivity.java

```diff
@@ -77,6 +80,7 @@ protected void onCreate(Bundle savedInstanceState) {
                 ((ImageView) findViewById(R.id.artImage)).setImageBitmap(bitmap);
                 displayArtInformation(artDescription);
                 postButton.setOnClickListener(v -> postImage(imageDownloadUrl, artDescription, List.of(artDescription.getCountry(), artDescription.getCity(), artDescription.getMuseum())));
+                shareButton.setOnClickListener(v -> shareImage(imageUri, artDescription));
             } catch (FileNotFoundException e) {
```

Comment:
Is the share button only available when we do a new scan ? If so could we make it always visible ?

Reply:
Yes it is only available on scan for people wanting to share their post outside the app instead of posting them. But we can make them always sharable. Can everyone share everything?

Reply:
Oh ok I get it. I think for now we can let it like that and we will see if we allow everyone to share posts at any moment.